src/parser-babylon.js

```diff
@@ -25,12 +26,14 @@ function parse(text) {
     ]
   };
 
+  const parseMethod = options.parseExpression ? "parseExpression" : "parse";
```

Comment:
@azz did something similar recently but different. Do you know if we can reuse the same thing?